README.md

```diff
@@ -2,133 +2,25 @@
-
-## Models
-
-The Cozy datastore stores documents, which can be seen as JSON objects. A `doctype` is simply a declaration of the fields in a given JSON object, to store similar objects in an homogeneous fashion.
-
-Cozy ships a [built-in list of `doctypes`][doctypes] for representation of most of the common documents (Bills, Contacts, Files, ...).
-
-Whenever your app needs to use a given `doctype`, you should:
-
-- Check if this is a standard `doctype` defined in Cozy itself. If this is the case, you should add a model declaration in your app containing at least the fields listed in the [main fields list for this `doctype`][doctypes]. Note that you can extend the Cozy-provided `doctype` with your own customs fields. This is typically what is done in [Konnectors] for the [Bill `doctype`][bill-doctype].
-- If no standards `doctypes` fit your needs, you should define your own `doctype` in your app. In this case, you do not have to put any field you want in your model, but you should crosscheck other cozy apps to try to homogeneize the names of your fields, so that your `doctype` data could be reused by other apps. This is typically the case for the [Konnector `doctype`][konnector-doctype] in [Konnectors].
-
-
-### Open a Pull-Request
-
-If you want to work on Home and submit code modifications, feel free to open pull-requests! See the [contributing guide][contribute] for more information about how to properly open pull-requests.
-
-
-Community
----------
-
-### Localization
-
-Localization and translations are handled by [Transifex][tx], which is used by all Cozy's apps.
-
-As a _translator_, you can login to [Transifex][tx-signin] (using your Github account) and claim an access to the [app repository][tx-app]. Locales are pulled when app is build before publishing.
-
-As a _developer_, you must [configure the transifex client][tx-client], and claim an access as _maintainer_ is the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
-
-
-### Maintainer
-
-The lead maintainer for Cozy Home is [Greg](https://github.com/gregorylegarec), send him/her a :beers: to say hello!
-
-
-### Get in touch
-
-You can reach the Cozy Community by:
-
-- Chatting with us on IRC [#cozycloud on Freenode][freenode]
-- Posting on our [Forum][forum]
-- Posting issues on the [Github repos][github]
-- Say Hi! on [Twitter][twitter]
+[Cozy][] is a platform that brings all your web services in the same private space. With it, your webapps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one's tracking you.
 
+## What is Home ?
 
-License
--------
+An application to configure and runs cozy konnectors. It is the main entrypoint to a Cozy.
 
-Cozy Home is developed by Cozy Cloud and distributed under the [AGPL v3 license][agpl-3.0].
```

Comment:
I know that we have a LICENCE.md file, but having this information in the readme is a quick access to it

Reply:
OK, I never use it, I re add it if you find it useful.

Reply:
Done